tidy3d/components/boundary.py

```diff
@@ -431,6 +454,10 @@ class StablePML(AbsorberSpec):
         description="'Stable' parameters of the complex frequency-shifted absorption poles.",
     )
 
+    _warn_num_layers = warn_num_layers_factory(
+        MIN_NUM_STABLE_PML_LAYERS, "stable perfectly-matched layer"
```

Comment:
in general we will pass arguments with their keywords, such as `fn(min_num_layers= MIN_NUM_STABLE_PML_LAYERS, ...)`. it is technically not required but makes it easier to read and reason about.